robot/impl/local_robot_test.go

```diff
@@ -3436,15 +3378,3 @@ func TestCloudMetadata(t *testing.T) {
 		})
 	})
 }
-
-//revive:disable-next-line:context-as-argument
-func initTestRobot(t *testing.T, ctx context.Context, cfg *config.Config, logger logging.Logger) (robot.LocalRobot, func()) {
-	t.Helper()
-
-	r, err := robotimpl.New(ctx, cfg, logger)
-	test.That(t, r, test.ShouldNotBeNil)
-	test.That(t, err, test.ShouldBeNil)
-	return r, func() {
-		test.That(t, r.Close(ctx), test.ShouldBeNil)
-	}
-}
```

Comment:
just a note that this helper served as the model the share helper I am adding in this PR - the only real difference is the use of `t.Cleanup` instead of returning a cleanup function